.github/workflows/linux_arm64.yml

```diff
@@ -0,0 +1,212 @@
+name: Linux ARM64 (Ubuntu 20.04, Python 3.11)
```

Comment:
let's state that it's `Linux ARM64 with Conan`

Reply:
I will address this and other comments (including those from other pipelines' PRs) in a separate PR.